### PR TITLE
(maint) Add documentation for the PuppetDB CLI configuration options

### PR DESCRIPTION
**Pull request by ajroetker**
This commit adds documentation for the CLI's configuration options
rather than just linking to the man pages.

### CODE REVIEW

File: documentation/pdb_client_tools.markdown [hunk 1]
```diff
@@ -48,11 +52,35 @@ certificate-whitelist and specify the paths to the CLI node's cacert, cert, and
...
+  Windows - C:/ProgramData/PuppetLabs/client-tools/ssl/certs/ca.pem
+  
+- `cert` An SSL certificate signed by your site's Puppet CA. Note that the PE
+ version of the CLI supports token auth via `puppet-access` and this option
+ should not be necessary.
+
+- `key` The private key for that certificate. Note that the PE version of the
+ CLI supports token auth via `puppet-access` and this option should not be
+ necessary.
+
```
Comment (ipeldridge) [inline documentation/pdb_client_tools.markdown]: In all the other config docs for various client tools, we list the options and then have a JSON example: https://docs.puppet.com/pe/latest/code_mgr_cli.html#installing-and-configuring-puppet-code-on-agents-and-workstations

File: documentation/pdb_client_tools.markdown [hunk 2]
```diff
@@ -6,11 +6,15 @@ layout: default
...
 [export]: ./anonymization.html
+[installpeclienttools]: /pe/latest/install_pe_client_tools.html#non-managed-workstation-installation
 
 # PuppetDB CLI
 
 ## Installation
 
+If you are installing `pe-client-tools` please see
+[the pe-client-tools installation instructions][installpeclienttools].
+
```
Comment (ipeldridge) [inline documentation/pdb_client_tools.markdown]: Do you want to specify here that you can install the client tools on a managed or non-managed workstation?

It's all covered in the installation docs: 

https://docs.puppet.com/pe/latest/install_pe_client_tools.html

**Comment by ajroetker** [comment]
@ipeldridge Added an example and a note about being able to install on managed or unmanaged!

File: documentation/pdb_client_tools.markdown [hunk 3]
```diff
@@ -48,11 +55,49 @@ certificate-whitelist and specify the paths to the CLI node's cacert, cert, and
 private key when using the CLI either with flags or a configuration file.
 
 To configure the PuppetDB CLI to talk to your PuppetDB with flags, add a
-configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf`. For more
+configuration file at `$HOME/.puppetlabs/client-tools/puppetdb.conf` (or
+`$USERPROFILE/.puppetlabs/client-tools/puppetdb.conf` for Windows). For more
```
Comment (johnduarte) [inline documentation/pdb_client_tools.markdown]: Should this be `%USERPROFILE%\.puppetlabs\client-tools\puppetdb.conf` for Windows notation?

File: documentation/pdb_client_tools.markdown [hunk 4]
```diff
@@ -48,11 +55,49 @@ certificate-whitelist and specify the paths to the CLI node's cacert, cert, and
...
+  urls) of your PuppetDB servers to query or manage via the CLI commands. (You
+  can set this with the `puppetdb_urls` parameter in the
+  puppet_enterprise::profile::controller class for PE.)
+
+  Default value: https://127.0.0.1:8080
+
+- `cacert` The path for the CA cert.
+
+  *nix sytems - /etc/puppetlabs/puppet/ssl/certs/ca.pem  
+  Windows - C:/ProgramData/PuppetLabs/client-tools/ssl/certs/ca.pem
```
Comment (johnduarte) [inline documentation/pdb_client_tools.markdown]: Do we use backslashes when referencing Windows paths?

Comment (ipeldridge) [inline documentation/pdb_client_tools.markdown]: Shouldn't it be C:\ProgramData\PuppetLabs\client-tools\ssl\certs\ca.pem?

Comment (ajroetker) [inline documentation/pdb_client_tools.markdown]: I can change it :) fwiw I copied this from the Orch docs :)

**Comment by ajroetker** [comment]
@ipeldridge @johnduarte Fixed the Windows docs!

File: documentation/pdb_client_tools.markdown (hunk 4 above)
Comment (ipeldridge) [inline documentation/pdb_client_tools.markdown]: hahahah. yeah I just changed it.

Comment (ipeldridge) [inline documentation/pdb_client_tools.markdown]: oh hey, @ajroetker I think the path should actually be C:\ProgramData\PuppetLabs\client-tools\ssl\certs\ca.pem if it's a managed workstation and C:\ProgramData\PuppetLabs\puppet\etc\ssl\certs\ca.pem is it's non-managed? @johnduarte ?

**Comment by johnduarte** [comment]
Thanks @ajroetker ! LGTM :+1:

**Comment by ajroetker** [comment]
@ipeldridge made the unmanaged/managed windows path changes

File: documentation/pdb_client_tools.markdown (hunk 4 above)
Comment (ipeldridge) [inline documentation/pdb_client_tools.markdown]: @ajroetker, sorry, but upon further review what I told you was wrong. take out the unmanaged and managed and just say this is the path to the ca cert: `C:\ProgramData\PuppetLabs\puppet\etc\ssl\certs\ca.pem`. I asked Nick Lewis to check the code.

**Pull request merged**